test/stresstest/broker.py

```diff
@@ -0,0 +1,42 @@
+########################################################################
+# Copyright (c) 2020 Robert Bosch GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+########################################################################
+
+import subprocess
+import config
+import signal
+
+class Broker:
+    def handleSIGINT(self, signum, _):
+        if self.process != 0:
+            os.kill(self.process, signal.SIGTERM)
+    def start(self):
+        port = self.cfg.get("port")
+        address = self.cfg.get("ip")
+        kuksa_val_databroker_cmd = ['cargo', 'run', '--release', '--bin', 'databroker', '--', '--metadata', '../../data/vss-core/vss_release_3.0.json', '--port', f"{port}", '--address', f"{address}"]
+        self.process = subprocess.check_call(kuksa_val_databroker_cmd, stdin=None, stdout=None, stderr=None)
```

Comment:
As databroker is expected to run indefinitely, `subprocess.check_call` will never return here.
If you start `python broker.py` and you then hit Ctrl-C,  `subprocess.check_call` will raise a `subprocess.CalledProcessError` which you don't catch here (which is fine as we **do** want to exit here anyway).
This means though `handleSIGINT` will never be called. As all that it does is terminating databroker's process anyway, I'd say we can get rid of custom signal handling here.

Reply:
`check_call` already set `stdin=None, stdout=None, stderr=None` by default so I think you don't need to set them here.